components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/pom.xml

```diff
@@ -169,6 +169,10 @@
             <artifactId>org.wso2.carbon.identity.organization.management.application</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.organization.management</groupId>
+            <artifactId>org.wso2.carbon.identity.organization.resource.sharing.policy.management</artifactId>
+        </dependency>
```

Comment:
Shouldn't this be in <scope>provided</scope> as this doesn't need to be packed in the jar ?